.github/workflows/pull_request.yml

```diff
@@ -53,6 +53,11 @@ jobs:
   validate-schemas:
     needs: build
     runs-on: ubuntu-latest
+    services:
+      validator:
+        image: onsdigital/eq-questionnaire-validator:custom-section-summaries
+        ports:
+          - 5001:5000
```

Comment:
We've removed the use of validator as a service on other repo's so that we only have to specify a branch change in only one place (run_validator). This is breaking on this repo at the moment because the run-validator already starts validator.